### DISCUSSION TITLE
How to add custom configuration such like on the  apiserver

### DISCUSSION THREAD

**Discussion by bingo4933** [rancher/rke2#4236] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
Hello, 
I've installed rke2 k8s cluster, which version info is:
```
$ rke2 -v
rke2 version v1.23.14+rke2r1 (83c8eb91b96ed5a2a3beb06c905e075724305a46)
go version go1.17.5b7
```

I'd like to check or enable configuration such like `--feature-gates=RemoveSelfLink=false` on api server

How to add  that?

**Describe the solution you'd like**
how to find the configuration file which I'd like to add custom setting for k8s component.

**Describe alternatives you've considered**

**Additional context**
Thanks a lot

Regards,

**Comment by brandond** [comment, marked as answer]
https://docs.rke2.io/reference/server_config

You're looking for `kube-apiserver-arg` for passing args to the apiserver. There are other options for other components, and additional options for configuring custom resource requests/requirements and adding additional environment variables and mount points to the static pods.
